types/index.d.ts

```diff
@@ -765,12 +765,20 @@ export interface EventComponentProps extends TranslationContextValue {
 
 export interface GalleryProps {
   images: Client.Attachment[];
+  t: i18next.TFunction;
 }
 
 export interface ImageProps {
-  image_url: string;
-  thumb_url: string;
-  fallback: string;
+  image_url?: string;
+  thumb_url?: string;
+  fallback?: string;
+}
+
+export interface ImageModalProps {
+  images: { src: string }[];
+  toggleModal: () => void;
+  index?: number;
```

Comment:
is it optional?

Reply:
yeah, in the Image component we don't provide